hbase/src/main/java/org/apache/metamodel/hbase/HBaseUpdateCallback.java

```diff
@@ -0,0 +1,137 @@
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.hbase;
+
+import java.io.IOException;
+
+import org.apache.metamodel.AbstractUpdateCallback;
+import org.apache.metamodel.UpdateCallback;
+import org.apache.metamodel.create.TableCreationBuilder;
+import org.apache.metamodel.delete.RowDeletionBuilder;
+import org.apache.metamodel.drop.TableDropBuilder;
+import org.apache.metamodel.insert.RowInsertionBuilder;
+import org.apache.metamodel.schema.MutableSchema;
+import org.apache.metamodel.schema.Schema;
+import org.apache.metamodel.schema.Table;
+
+public class HBaseUpdateCallback extends AbstractUpdateCallback implements UpdateCallback {
+
+    private final HBaseConfiguration _configuration;
+
+    private final HBaseDataContext _dataContext;
+
+    public HBaseUpdateCallback(HBaseDataContext dataContext) {
+        super(dataContext);
+        _configuration = dataContext.getConfiguration();
+        _dataContext = dataContext;
+    }
+
+    @Override
+    public TableCreationBuilder createTable(Schema schema, String name) throws IllegalArgumentException,
+            IllegalStateException {
+        throw new UnsupportedOperationException(
+                "Use createTable(Schema schema, String name, HBaseColumn[] outputColumns)");
+    }
+
+    public HBaseCreateTableBuilder createTable(Schema schema, String name, HBaseColumn[] outputColumns)
+            throws IllegalArgumentException, IllegalStateException {
+        return new HBaseCreateTableBuilder(this, schema, name, outputColumns);
+    }
+
+    @Override
+    public boolean isDropTableSupported() {
+        return true;
+    }
+
+    @Override
+    public TableDropBuilder dropTable(Table table) throws IllegalArgumentException, IllegalStateException,
+            UnsupportedOperationException {
+        return new HBaseTableDropBuilder(table, this);
+    }
+
+    public void dropTableExecute(Table table) {
+        try {
+            final HBaseWriter HbaseWriter = new HBaseWriter(HBaseDataContext.createConfig(_configuration));
+            HbaseWriter.dropTable(table.getName());
+            MutableSchema schema = (MutableSchema) table.getSchema();
+            schema.removeTable(table);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Can you remove this and instead throw a `MetaModelException` which wraps the `IOException`?